src/Database/DBStructure.php

```diff
@@ -1299,6 +1299,16 @@ public static function definition() {
 						"created" => array("created"),
 						)
 				);
+		$database["participation"] = array(
+				"fields" => array(
+						"item" => array("type" => "int(10) unsigned", "not null" => "1", "primary" => "1", "relation" => array("item" => "id")),
```

Comment:
This is a departure from the current foreign key naming scheme. So far, a foreign key to `item.id` has been named `iid` and a foreign key to `contact.id` has been named `cid` or the distinct purpose, like `author` and `owner`.

Is this intended? I'd rather keep the same kind of naming everywhere in the database.

Reply:
Yeah, this is the style at my job. Sorry.